src/DynamoCore/Graph/Nodes/NodeModelExtensions.cs

```diff
@@ -127,7 +127,7 @@ internal static List<IGraphicItem> GeneratedGraphicItems(this NodeModel node, En
         /// </summary>
         /// <param name="node">target NodeModel</param>
         /// <returns>Original node name as string</returns>
-        internal static string GetOriginalName(this NodeModel node)
+        public static string GetOriginalName(this NodeModel node)
```

Comment:
I'd prefer this stay internal, it's kind of a mess, maybe just give the extension internals access with `internalVisibleToAttribute`

Reply:
Done! Thank you for that point, I was not comfortable exposing this method, but I wasn't aware of the internalVisibleToAttribute class. The change was reverted and I implemented the internalVisibleToAttribute for this assembly, all working great, thanks a ton!